src/CodeGen_C.cpp

```diff
@@ -315,6 +315,29 @@ class TypeInfoGatherer : public IRGraphVisitor {
     std::set<Type> vector_types_used;
 };
 
+std::string maybe_emit_numeric_constant(const Expr &e) {
```

Comment:
Should this use the word "emit" if it is just returning the text, not emitting it to the output?

Reply:
Fair, will fix